wpilibc/athena/src/DriverStation.cpp

```diff
@@ -584,9 +582,13 @@ void DriverStation::Run() {
   while (m_isRunning) {
     HAL_WaitForDSData();
     GetData();
+    // notify IsNewControlData variables
+    m_newControlData = true;
```

Comment:
If it's not done inside the block, then the guarantee I mention in #233 won't hold (namely, that isNewControlData will always return true when waitForData returns).